contxt/services/iot.py

```diff
@@ -114,6 +125,79 @@ def get_time_series_for_field(
             per_page=per_page,
         )
 
+    def get_time_series_for_fields(
```

Comment:
Why not combine get_time_series_for_field and this new method to just take a list of field[s]?

Or keep the method signature the same for the singular method and just call the new method so you don't split logic?

Reply:
I could, but the behavior and return type is pretty different in those cases.

1. `get_time_series_for_field`: returns one paginated time series (i.e. the time series is lazily loaded)
2. `get_time_series_for_field`: returns a list of non-paginated time series (i.e. the pages are loaded up front via batch requests)

The caller may prefer the behavior of one over the other depending on what they need to do.

Reply:
Ok, i just wanted to see if there was a way to consolidate.  Will the caller be confused with the very similar names or do you think the pydoc will ensure they are calling what is needed?